### ISSUE TITLE
[Windows Terminal - Settings - Color Schemes]: 'Add new' button's text is getting truncated and not visible properly at 200% text scaling.

### ISSUE THREAD

**Issue by SaurabhNew** [microsoft/terminal#17897]
### Windows Terminal version

1.22.2362.0

### Windows build number

27695.1000

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 27695.1000)
App: Windows Terminal Preview

### Steps to reproduce

**Pre-requisite:**
Settings>Display>Display Resolution (1680*1050)>set scaling to 100%>Increase the text slider to 200% >Apply>Reopen the application.

**Repro steps:**
1. Open the 'windows terminal' application.
2. Follow the above-mentioned pre-requisite step.
3. Now open 'Settings'.
4. Navigate to the 'Color Schemes' tab and activate it.
5. Navigate to the 'Add new' button. 
6. Observe the issue.

**User experience:**
Users with low vision rely on zooming features for navigation will feel difficulty in understanding the button's name, if name gets truncated at 200% zoom.

**WCAG Reference Link:**
https://www.w3.org/WAI/WCAG21/Understanding/resize-text

**Attachment:**
[MAS1.4.4 - Add new button's text is not visble properly at 200% text scaling.zip](https://github.com/user-attachments/files/16958099/MAS1.4.4.-.Add.new.button.s.text.is.not.visble.properly.at.200.text.scaling.zip)

### Expected Behavior

'Add new' button's text should be properly visible at 200% text scaling.

### Actual Behavior

'Add new' button's text is getting truncated and not visible properly at 200% text scaling.

**Note:**
Same issue repro for 'Foreground colors' & 'Set as default' button. (Navigation Path: Windows Terminal> Settings> Color schemes> Add new> 'Foreground colors' & 'Set as default button').

**Comment by carlos-zamora** [comment]
@SaurabhNew is there an associated severity rating for this issue?

**Comment by SaurabhNew** [comment]
@carlos-zamora,
Tags are updated now?
